projects/python/perception/object_detection_2d/nanodet/export_torchscript.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2020-2022 OpenDR European Project
```

Comment:
`export_onnx.py` and `export_torchscript.py` are almost identical. They can be merged in an `export_demo.py` and add an argparse argument for optimization method. It should not complicate stuff and choosing between the two optimization methods is mentioned plenty of times across the docs.

Another option would be to merge them into the inference method, as again they are almost the same. This would be similar to other tools' demos.